packages/utils/web-multi-logger/src/Logger.ts

```diff
@@ -49,16 +49,16 @@ export default class Logger {
    *
    * @returns the port number
    */
-  start(): Promise<number> {
-    return new Promise<number>((resolve) => {
-      this.server = this.app.listen(this.port, () => resolve(this.port));
-    });
+  async startPort(loggerPort: number): Promise<number> {
```

Comment:
We should clean up `this.port` in the constructor if it isn't being used anymore.